zeppelin-web/src/app/search/result-list.controller.js

```diff
@@ -74,12 +74,21 @@ angular
         };
       }
 
-      var lines = note.snippet
+      var result = '';
+      if (note.header !== '') {
+        result = note.header + '\n\n' + note.snippet;
+      }
+      else {
```

Comment:
To be consistent with the [rest of the project](https://github.com/ravicodder/incubator-zeppelin/blob/searchTitle/zeppelin-web/src/app/interpreter/interpreter.controller.js#L67) we better move `else` one line up gere and below in this file